client/index.jsx

```diff
@@ -7,23 +7,70 @@ class App extends React.Component {
 
     this.state = {
       currentGame: 0,
-      reviews: []
+      reviews: [{
+        gameId: 0,
+        date: Date.now,
+        overall: 1,
+        title: '',
+        review: '',
+        recommend: false,
+        nickname: '',
+        location: '',
+        email: '',
+        buyForSelf: false,
+        ageBracket: 1,
+        gender: 1,
+        graphics: 1,
+        gameplay: 1,
+        appeal: 1,
+        ownershipBracket: 1,
+        purchaseOnline: false,
+        readReviews: false,
+        recommendBGS: 1,
+        meta: {
+          helpful: 1,
+          unhelpful: 1,
+        },
+      }],
```

Comment:
What is the purpose of making this an array with one object and not just an object?

Reply:
There might be a more simple way of handling it, but the idea is that `ReviewList` is expecting an array of reviews to `map` over, so I put in an array with one empty record to keep any errors from occurring until the first `GET` in `App`'s `componentDidMount` has time to bring back the real data.